modules/sovrnBidAdapter.js

```diff
@@ -22,48 +23,56 @@ export const spec = {
    * @return object of parameters for Prebid AJAX request
    */
   buildRequests: function(bidReqs, bidderRequest) {
-    const loc = utils.getTopWindowLocation();
-    let sovrnImps = [];
-    let iv;
-    utils._each(bidReqs, function (bid) {
-      iv = iv || utils.getBidIdParameter('iv', bid.params);
-      sovrnImps.push({
-        id: bid.bidId,
-        banner: { w: 1, h: 1 },
-        tagid: String(utils.getBidIdParameter('tagid', bid.params)),
-        bidfloor: utils.getBidIdParameter('bidfloor', bid.params)
+    try {
+      const loc = utils.getTopWindowLocation();
+      let sovrnImps = [];
+      let iv;
+      utils._each(bidReqs, function (bid) {
+        iv = iv || utils.getBidIdParameter('iv', bid.params);
+        sovrnImps.push({
+          id: bid.bidId,
+          banner: {
+            format: bid.sizes.map(size => { return {w: size[0], h: size[1]} }),
```

Comment:
does javascript let you turn this into `....map(size => {w: size[0], h: size[1]} )`?

Reply:
also, looking at the `transformSizes()` function in the `appnexusBidAdapter`, it looks like `bid.sizes` can take different forms, maybe we should account for that as well?

Reply:
#1 you can use ({..}) but if you start with a curly brace then it thinks you are starting a multiline function.
#2 Good catch, will look into and update.

Reply:
never mind, it is possible to get a single array as the sizes object - will address.